### DISCUSSION TITLE
What is the .NET 8.0 Blazor fix for <base href="~/" />?

### DISCUSSION THREAD

**Discussion by mxmissile** [dotnet/aspnetcore#53570] [Q&A]
Adding that to my Blazor Server app would allow it to run the wherever I deployed it, root, subfolder, within VS etc etc. It would just work without configuration or hassle.  How do I add this functionality back? I do not know where the app will be deployed at designtime.

**Comment by Xyncgas** [comment]
I believe you should find "./_content/{RazorProject's ProjectName in visual studio}/{filename at wwwroot}"
